### PR TITLE
Main E2E

### PR DESCRIPTION
**Pull request by jhansitavva**

### CODE REVIEW

File: app/src/main/java/com/example/sports/ui/theme/Theme.kt [hunk 1]
```diff
@@ -126,3 +126,23 @@ fun SportsTheme(
...
+        Build.VERSION.SDK_INT >= 29 -> Color.Transparent.toArgb()
+        Build.VERSION.SDK_INT >= 26 -> Color(0xFF, 0xFF, 0xFF, 0x63).toArgb()
+        // Min sdk version for this app is 24, this block is for SDK versions 24 and 25
+        else -> Color(0x00,0x00, 0x00, 0x50).toArgb()
+    }
+    window.navigationBarColor = navigationBarColor
+    val controller = WindowCompat.getInsetsController(window, view)
+    controller.isAppearanceLightStatusBars = !darkTheme
+    controller.isAppearanceLightNavigationBars = !darkTheme
+}
```
Comment (johnshea) [inline app/src/main/java/com/example/sports/ui/theme/Theme.kt]: Add newline at EOF.

File: app/build.gradle.kts
Comment (johnshea) [inline app/build.gradle.kts:1]: Upgrade BOM to `2023.08.00`.

**Review by johnshea — CHANGES_REQUESTED** [review]
Navigation bar is not translucent and it should be.
Screenshot 2023-08-14 at 9 50 04 AM

Update gradle wrapper to `8.2`

**Review by johnshea — CHANGES_REQUESTED** [review]
UI does not have translucent navigation bar on lower APIs but it should.

Update gradle wrapper to `8.2`


Screenshot 2023-08-17 at 2 06 30 AM

**Comment by jhansitavva** [comment]
> UI does not have translucent navigation bar on lower APIs but it should.
> 
> Update gradle wrapper to `8.2`
> 
> Screenshot 2023-08-17 at 2 06 30 AM

Navbar is translucent, it is just the padding that you are noticing.

File: app/src/main/java/com/example/sports/ui/SportsScreens.kt [hunk 2]
```diff
@@ -353,26 +372,32 @@ private fun SportsListAndDetail(
...
-            modifier = Modifier.weight(2f),
-            onClick = onClick
+            onClick = onClick,
+            contentPadding = contentPadding,
+            modifier = Modifier
+                .weight(2f)
+                .padding(horizontal = dimensionResource(R.dimen.padding_medium))
         )
-        val activity = (LocalContext.current as Activity)
+        //val activity = (LocalContext.current as Activity)
```
Comment (johnshea) [inline app/src/main/java/com/example/sports/ui/SportsScreens.kt]: Remove commented out code.

**Review by johnshea — APPROVED** [review]
LGTM with small changes.

- Update gradle wrapper to `8.2`
- Remove commented out code

**Pull request merged**